codeworld-server/src/Build.hs

```diff
@@ -22,14 +22,17 @@ module Build where
 import           Control.Monad
 import           Data.ByteString (ByteString)
 import qualified Data.ByteString as B
+import qualified Data.ByteString.Char8 as C
 import           System.Directory
 import           System.FilePath
 import           System.IO
 import           System.IO.Temp (withSystemTempDirectory)
 import           System.Process
 import           Text.Regex.TDFA
+import           Text.Regex
```

Comment:
Is this necessary?  I don't see that you've added any uses of it in this module.